schemas/Identifier.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Identifier",
-  "description": "An identifier for a user or institution.",
+  "description": "An identifier for a user or institution. Between 2-20 characters inclusively. May include dashes (-), but not at the beginning or end and no more than one consecutively.",
```

Comment:
That makes sense to me, but just to test our reasoning a bit:

* Why would we need UUID as *usernames*?
* Also keep in mind that this schema is easier to relax than tighten - once you have a user with a certain name, you can't easily get rid of them, but if you want to roll out a patch that allows more different names, that's fine.
* Maybe there is a reason why the ledger might want to use UUID looking usernames *itself*? Related example: If we have usernames of up to 60 characters, we can no longer add a feature where you can send money to an Ed25519 public key and the ledger will hold it until someone with that key shows up to claim it - because an attacker could sneakily make an account with that name. Bit contrived, but I'm just advocating for the devil.